release.sh

```diff
@@ -114,19 +117,20 @@ rm -f package.json.bak
 
 # Update the version number in CDN URLs included in the README files,
 # and regenerate the Subresource Integrity hash for these files.
-node update-sri.js "${VERSION}" README.md contrib/*/README.md
+node update-sri.js "${VERSION}" README.md contrib/*/README.md dist/README.md
 
 # Make the commit and tag, and push them.
-git add package.json bower.json
+git add package.json bower.json README.md contrib/*/README.md dist/README.md
 git commit -n -m "v$VERSION"
+git diff --stat --exit-status # check for uncommitted changes
```

Comment:
That's the idea, yes. We have an `set -e -o pipefail` in line 3 to achieve that. One benefit is that since this is a `git` command invocation, it will be printed instead of executed in dry run mode, so we can continue there even if the check for uncommitted changes up front (the one with `command git`) fails.